sci-biology/abyss/abyss-2.0.2-r1.ebuild

```diff
@@ -51,5 +59,14 @@ src_configure() {
 	# unless request by user: bug #534412
 	use misc-haskell || export ac_cv_prog_ac_ct_GHC=
 
-	econf $(use_enable openmp)
+	# bug #646478 , also where is mpich2 switch?
+	#  --enable-mpich          use MPICH (default is to use Open MPI)
+	#  --enable-lammpi         use LAM/MPI (default is to use Open MPI)
+	econf $(use_enable openmp) \
+		$(use_enable mpich) \
+		$(use_with sqlite) \
+		$(use_with mpi) \
```

Comment:
You say USE=-mpi does not really work. ./configure-based systems frequently allow you override autodetection even thing is not exposed as a flag. In this case setting `ac_cv_header_mpi_h=no` makes it believe mpi.h is not present.